.github/ISSUE_TEMPLATE/Emulation_bug_report.yaml

```diff
@@ -0,0 +1,76 @@
+# Docs - https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+name: Bug Report
+description: Report a issue with the emulation (ie. graphical issue, crashes, unintended behavior with doesn't occur on hardware).
+title: "[BUG]: "
+labels: [Bug]
```

Comment:
Existing label is `bug`, not `[Bug]`. https://github.com/cemu-project/Cemu/issues?q=label%3Abug